docs/src/dev/how_it_works.md

```diff
@@ -102,24 +102,28 @@ two calls to ``f`` to evaluate the gradient:
                x_3 \\
                x_4
            \end{bmatrix}
-
-\vec{x}_{\epsilon} = \begin{bmatrix}
+```
+1st call:
+```math
+\to \vec{x}_{\epsilon} = \begin{bmatrix}
                         x_1 + \epsilon_1 \\
                         x_2 + \epsilon_2 \\
                         x_3 \\
                         x_4
                      \end{bmatrix}
 \to
```

Comment:
Minor point, but can I suggest fixing the arrows?
```suggestion
\vec{x}_{\epsilon} = \begin{bmatrix}
                        x_1 + \epsilon_1 \\
                        x_2 + \epsilon_2 \\
                        x_3 \\
                        x_4
                     \end{bmatrix}
\mapsto
```
IMO it's not ideal to use `\to` with two different nonstandard meanings.